starter-code/src/clue.js

```diff
@@ -92,3 +181,63 @@ var roomsArray = [];
 
 // Weapons Collection
 var weaponsArray = [];
+
+// Push characters
+charactersArray.push(mrGreen);
+charactersArray.push(drOrchid);
+charactersArray.push(profPlum);
+charactersArray.push(missScarlet);
+charactersArray.push(mrsPeacock);
+charactersArray.push(mrMustard);
```

Comment:
You can also psuh them all at once doing:
```
 charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsScarlet, mrsPeacock, mrMustard)
```

Reply:
thx, i didn't know!